account_asset_management/models/account_asset.py

```diff
@@ -563,6 +563,21 @@ def group_lines(x, y):
                 amount_diff = round(
                     residual_amount_table - residual_amount, digits)
                 if amount_diff:
+                    # auto-create new line only if is needed
+                    total_table_lines = 0
+                    for table_i, entry in enumerate(table):
```

Comment:
`table_i` not used, but why don't you use `mapped` and do everything in one line?